### PR TITLE
fetch/

### PR DESCRIPTION
**Pull request by jeesupark96**
image

### CODE REVIEW

File: fetch/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const fetchResponse = fetch('https://jsonplaceholder.typicode.com/users', {
+  method: 'GET'
+})
+  .then(response => response.json(fetchResponse))
+  .catch(console.error);
+console.log(fetchResponse);
```
Comment (scbowler) [inline fetch/main.js]: Although it looks like this is working it is not. You need a second `.then` to get your data. Should look like:

```JAVASCRIPT
fetch('https://jsonplaceholder.typicode.com/users', {
  method: 'GET'
})
  .then(response => response.json(fetchResponse))
  .then(data => console.log(data))
  .catch(console.error);
```

File: fetch/main.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+const pokemonResponse = fetch('https://pokeapi.co/api/v2/pokemon/4', {
+  method: 'GET'
+})
+  .then(response => response.json(pokemonResponse))
+  .catch(console.error);
+console.log([pokemonResponse]);
```
Comment (scbowler) [inline fetch/main.js]: Same issue with this one

**Review by scbowler — CHANGES_REQUESTED** [review]
A couple updates needed. Let me know if you have any questions.

**Comment by jeesupark96** [comment]
sounds good ! thank you

**Review by scbowler — APPROVED** [review]
Much better! Looks good! 👍

**Pull request merged**